--- /usr/lib/node_modules/cozy-controller/build/server/lib/user.js.orig	2015-03-02 12:06:51.338138849 +0000
+++ /usr/lib/node_modules/cozy-controller/build/server/lib/user.js	2015-03-02 12:04:09.028303046 +0000
@@ -18,6 +18,8 @@
   env = {};
   user = env.USER = app.user;
   appdir = env.HOME = config('dir_source');
+  env.SHELL = process.env.SHELL;
+  env.PATH = process.env.PATH;
   child = spawn('bash', [path.join(__dirname, '..', 'lib', 'adduser.sh')], {
     env: env
   });
